### PR TITLE
Feature/birch style

### LINKED ISSUE #1614: Cosmetic Issue
When you have a build queue that is long.  Your gui doesn't handle it correctly.

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #1614

# Changes proposed in this pull request:
- Fixes a visual bug with the list of build orders
- Adds the Birch style

Review please

### CODE REVIEW

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 32 issues
* ![MINOR][MINOR] 32 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [blocks#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/blocks#): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/client/gui#): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/client/render#): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/colony#): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/colony/buildings#): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/colony/permissions#): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [requestsystem#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/colony/requestsystem#): Add a 'package-info.java' file to document the 'requestsystem' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [init#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/colony/requestsystem/init#): Add a 'package-info.java' file to document the 'init' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [locations#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/colony/requestsystem/locations#): Add a 'package-info.java' file to document the 'locations' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [requests#](https://github.com/Minecolonies/minecolonies/blob/cbaa2415aa9338045c0debad363aa0c0c96df109/src/main/java/com/minecolonies/coremod/colony/requestsystem/requests#): Add a 'package-info.java' file to document the 'requests' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Asherslab** [comment]
Looks good to me! :shipit:

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1631/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

File: src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBarracks.java [hunk 1]
```diff
@@ -112,21 +113,24 @@ public void onUpgradeComplete(final int newLevel)
         }
 
         EnumFacing facing = EnumFacing.NORTH;
+
+        final int offset = getStyle().toLowerCase(Locale.ENGLISH).contains("birch") ? TOWER_OFFSET + 1: TOWER_OFFSET;
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBarracks.java:117]: MAJOR Add parentheses to make the operator precedence explicit. [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS864)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 33 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 32 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [blocks#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/blocks#): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/client/gui#): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/client/render#): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/colony#): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/colony/buildings#): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/colony/permissions#): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [requestsystem#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/colony/requestsystem#): Add a 'package-info.java' file to document the 'requestsystem' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [init#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/colony/requestsystem/init#): Add a 'package-info.java' file to document the 'init' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [locations#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/colony/requestsystem/locations#): Add a 'package-info.java' file to document the 'locations' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [requests#](https://github.com/Minecolonies/minecolonies/blob/5b1aa1d6cde527c16a2410561445b9a411d948ba/src/main/java/com/minecolonies/coremod/colony/requestsystem/requests#): Add a 'package-info.java' file to document the 'requests' package [rule](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**